cmd/kubeadm/app/apis/kubeadm/v1beta3/types.go

```diff
@@ -28,6 +28,9 @@ import (
 type InitConfiguration struct {
 	metav1.TypeMeta `json:",inline"`
 
+	// +optional
```

Comment:
[1] I think we should document in the API that this field in this field - in this context - exists only for making our types kustomize friendly

Another option will be to use this field to express the node name, but I'm not entirely sure this makes really sense

Reply:
This should be clarified before merge